webapp/components/features/FiledReportsTable/FiledReportsTable.vue

```diff
@@ -1,73 +1,38 @@
 <template>
-  <ds-table
-    class="nested-table"
-    v-if="filed && filed.length"
-    :data="filed"
-    :fields="fields"
-    condensed
-  >
-    <template #submitter="scope">
-      <user-teaser
-        :user="scope.row.submitter"
-        :showAvatar="false"
-        :showPopover="false"
-        data-test="filing-user"
-      />
-    </template>
-    <template #reportedOn="scope">
-      <ds-text size="small">
-        <hc-relative-date-time :date-time="scope.row.createdAt" data-test="filed-date" />
-      </ds-text>
-    </template>
-    <template #reasonCategory="scope">
-      {{ $t('report.reason.category.options.' + scope.row.reasonCategory) }}
-    </template>
-    <template #reasonDescription="scope">
-      {{ scope.row.reasonDescription.length ? scope.row.reasonDescription : '—' }}
-    </template>
-  </ds-table>
+  <ds-card v-if="filedReports && filedReports.length > 0">
+    <ds-table :data="filedReports" :fields="fields" condensed>
+      <template #resource="scope">
+        <filed-report :filed-report="scope.row" />
+      </template>
+    </ds-table>
+  </ds-card>
+  <ds-card v-else>
+    <ds-space>
+      <ds-placeholder>
+        {{ $t('settings.filed-reports.empty') }}
+      </ds-placeholder>
+    </ds-space>
+  </ds-card>
 </template>
+
 <script>
-import UserTeaser from '~/components/UserTeaser/UserTeaser'
-import HcRelativeDateTime from '~/components/RelativeDateTime'
+import FiledReport from '~/components/features/FiledReport/FiledReport'
 
 export default {
   components: {
-    UserTeaser,
-    HcRelativeDateTime,
+    FiledReport,
   },
   props: {
-    filed: { type: Array, default: () => [] },
+    filedReports: { type: Array, default: () => [] },
   },
   computed: {
     fields() {
       return {
-        submitter: {
-          label: this.$t('moderation.reports.submitter'),
-          width: '15%',
-        },
-        reportedOn: {
-          label: this.$t('moderation.reports.reportedOn'),
-          width: '20%',
-        },
-        reasonCategory: {
-          label: this.$t('moderation.reports.reasonCategory'),
-          width: '30%',
-        },
-        reasonDescription: {
-          label: this.$t('moderation.reports.reasonDescription'),
-          width: '35%',
-        },
+        resource: this.$t('settings.filed-reports.columns.resource'),
+        reasonDescription: this.$t('settings.filed-reports.columns.reasonDescription'),
+        reasonCategory: this.$t('settings.filed-reports.columns.reasonCategory'),
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-13T08:50:38Z" title="Thursday, February 13th 2020, 9:50:38 am +01:00">Feb 13, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,73 +1,38 @@
 <template>
-  <ds-table
-    class="nested-table"
-    v-if="filed && filed.length"
-    :data="filed"
-    :fields="fields"
-    condensed
-  >
-    <template #submitter="scope">
-      <user-teaser
-        :user="scope.row.submitter"
-        :showAvatar="false"
-        :showPopover="false"
-        data-test="filing-user"
-      />
-    </template>
-    <template #reportedOn="scope">
-      <ds-text size="small">
-        <hc-relative-date-time :date-time="scope.row.createdAt" data-test="filed-date" />
-      </ds-text>
-    </template>
-    <template #reasonCategory="scope">
-      {{ $t('report.reason.category.options.' + scope.row.reasonCategory) }}
-    </template>
-    <template #reasonDescription="scope">
-      {{ scope.row.reasonDescription.length ? scope.row.reasonDescription : '—' }}
-    </template>
-  </ds-table>
+  <ds-card v-if="filedReports && filedReports.length > 0">
+    <ds-table :data="filedReports" :fields="fields" condensed>
+      <template #resource="scope">
+        <filed-report :filed-report="scope.row" />
+      </template>
+    </ds-table>
+  </ds-card>
+  <ds-card v-else>
+    <ds-space>
+      <ds-placeholder>
+        {{ $t('settings.filed-reports.empty') }}
+      </ds-placeholder>
+    </ds-space>
+  </ds-card>
 </template>
+
 <script>
-import UserTeaser from '~/components/UserTeaser/UserTeaser'
-import HcRelativeDateTime from '~/components/RelativeDateTime'
+import FiledReport from '~/components/features/FiledReport/FiledReport'
 
 export default {
   components: {
-    UserTeaser,
-    HcRelativeDateTime,
+    FiledReport,
   },
   props: {
-    filed: { type: Array, default: () => [] },
+    filedReports: { type: Array, default: () => [] },
   },
   computed: {
     fields() {
       return {
-        submitter: {
-          label: this.$t('moderation.reports.submitter'),
-          width: '15%',
-        },
-        reportedOn: {
-          label: this.$t('moderation.reports.reportedOn'),
-          width: '20%',
-        },
-        reasonCategory: {
-          label: this.$t('moderation.reports.reasonCategory'),
-          width: '30%',
-        },
-        reasonDescription: {
-          label: this.$t('moderation.reports.reasonDescription'),
-          width: '35%',
-        },
+        resource: this.$t('settings.filed-reports.columns.resource'),
+        reasonDescription: this.$t('settings.filed-reports.columns.reasonDescription'),
+        reasonCategory: this.$t('settings.filed-reports.columns.reasonCategory'),
```


I would switch the order of the last two to have it the same like in other places.